app/models/auctify/sale/auction.rb

```diff
@@ -249,6 +263,23 @@ def auction_prolonging_limit_in_seconds
         pack&.auction_prolonging_limit_in_seconds || Auctify.configuration.auction_prolonging_limit_in_seconds
       end
 
+      def close_manually(by:, price_check:)
+        if manually_closed_at.nil? && update(manually_closed_at: Time.current, manually_closed_by: by, manually_closed_price_check: price_check)
+          Auctify::BiddingCloserJob.perform_later(auction_id: id)
+          true
+        else
+          false
+        end
+      end
+
+      def lock_bidding(by:)
```

Comment:
Tady budu potřebovat objasnit jaký je rozdíl/použití `auction.lick_bidding` a `auction.bidding_closed`. Respektive kdy dochází k `unlock_bidding`

Reply:
`lock_bidding` a `unlock_bidding` jsou adminem rucne volane akce z frontendu, ktere slouzi pro mezikrok kontroly dosazene ceny oproti salu pred samotnym manualnim uzavrenim

![image](https://user-images.githubusercontent.com/1286942/158773007-721665b1-44c7-4d31-a304-0690f4bac3b6.png)

![image](https://user-images.githubusercontent.com/1286942/158773164-afd8b03c-6550-4292-b151-423dbbe8c8e6.png)